app/models/question.rb

```diff
@@ -20,4 +20,8 @@ def add_tags(tag_string)
     self.tags = tag_array
   end
 
+  def unchosen_answers
+    self.answers.order('votes_count desc').select { |ans| ans != self.answer }
+  end
```

Comment:
Try do this in ActiveRecord, without using a Ruby select.  Maybe a where clause before the order method, maybe like:

self.where.not (id: self.answer.id).order . . .